src/components/Footer/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Logo } from '../../icons';
+import Separator from '../Separator';
+import { Text } from '../Text';
+import { TextWrapper, Wrapper } from './style';
+
+const Footer: React.FC = () => {
+  return (
+    <Wrapper>
+      <Logo brand="nightwatch" />
+      <Separator />
+      <Logo brand="browserstack" />
+      <TextWrapper>
+        <Text fontSize="--font-size-14" lineHight="--line-height-16" color="--color-grey-30">
```

Comment:
Can we move this to component-specific styles file?

Reply:
It's a component, not a style. I am passing data as a prop.